bitnami/ghost/templates/secrets.yaml

```diff
@@ -22,9 +22,23 @@ data:
   ghost-password: {{ randAlphaNum 10 | b64enc | quote }}
   {{- end }}
   {{- end }}
-  {{- if and .Values.smtpPassword (not .Values.smtpExistingSecret) }}
+---
+{{- end }}
+
+{{- if not .Values.smtpExistingSecret }}
```

Comment:
you need to check here the smtpPassword, you are creating an empty secret without data in this case.